pkg/controllers/vmsetcontroller/vmsetcontroller.go

```diff
@@ -224,7 +224,7 @@ func (v *VirtualMachineSetController) reconcileVirtualMachineSet(vmset *hfv1.Vir
 		glog.V(5).Infof("provisioning %d vms", needed)
 		// this code is so... verbose...
 		for i := 0; i < needed; i++ {
-			vmName := strings.Join([]string{vmset.Spec.BaseName, fmt.Sprintf("%08x", rand.Uint32())}, "-")
+			vmName := strings.Join([]string{"hobbyfarm", vmset.Spec.BaseName, fmt.Sprintf("%08x", rand.Uint32())}, "-")
```

Comment:
The purpose of the `vmset.Spec.BaseName` is to provide the ability to customize the VM name -- I don't think forcing `hobbyfarm` to the prefix is really necessary in this case, if you want `hobbyfarm` in the front then you can just set the VMSet base name to have `hobbyfarm-`

Reply:
Ah, I didn't realize that `vmset.Spec.BaseNam` was exposed because I didn't create a `VirtualMachineSet`, only a `VirtualMachineTemplate`.